src/visGeometry/GeometryStore.ts

```diff
@@ -378,6 +384,18 @@ class GeometryStore {
         });
     }
 
+    private async fetchVolume(url: string): Promise<VolumeModel> {
+        // TODO should this be in a worker? Are we already in a worker here?
+        //   Should this class get a `VolumeLoaderContext` going?
+        const model = new VolumeModel();
+        this.setGeometryInRegistry(url, model, GeometryDisplayType.VOLUME);
+        const loader = await createVolumeLoader(url);
+        // TODO onChannelLoaded callback?
+        const volume = await loader.createVolume(new LoadSpec());
+        model.setImage(volume);
+        return model;
+    }
+
```

Comment:
Not sure if this relevant, but the caching changes in review are removing the need for workers in that part of the code base, and there was discussion of workers affecting builds and publishing with NBSV.

Just a thought that we should use them if we need them, but if achieve 0 workers (automation fears??) then there may be some arguments for not adding them back in?

Reply:
unless I am forgetting, doesn't PDB geometry still use a worker to calculate levels of detail?

Reply:
The context here may be slightly different. In volume-viewer we moved loading to a separate worker because playing volume time series requires loading and processing quite a lot of data per frame, and it was a problem when that processing blocked rendering. We have an externally available type, called `VolumeLoaderContext`, for handling offloading that processing to a worker. Questions are

1. is it as worth it to use `VolumeLoaderContext` here as it is in volume-viewer, given your concern and that simulation volumes may be smaller on average than microscope volumes; and
3. if 1, who should own `VolumeLoaderContext` and how should it grant access to it?

To be resolved by future work.